### ISSUE TITLE
ExtPos on `rather than`

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_English-GUM#87]
The phrase `rather than` is typically parsed as a `fixed` expression in GUM, but without an `ExpPos`.  In EWT, it almost always has `ExpPos=CCONJ`, except for one instance where it is used before a clause in 

```
12      rather  rather  ADP     IN      ExtPos=SCONJ    14      mark    14:mark _
13      than    than    SCONJ   IN      _       12      fixed   12:fixed        _
14      undermining     undermine       VERB    VBG     Tense=Pres|VerbForm=Part        22      advcl   22:advcl:rather_than    _
15-16   Afghanistan’s   _       _       _       _       _       _       _       _
15      Afghanistan     Afghanistan     PROPN   NNP     Number=Sing     17      nmod:poss       17:nmod:poss    _
16      ’s      's      PART    POS     _       15      case    15:case _
17      stability       stability       NOUN    NN      Number=Sing     14      obj     14:obj  _
```

**Comment by amir-zeldes** [comment]
Sure, I can fix this.

**Comment by amir-zeldes** [comment]
Will be fixed upstream via ad7f8f0
